test/quickcheck/main.cc

```diff
@@ -0,0 +1,23 @@
+/**
+ * This file is part of the CernVM File System.
+ */
+
+#include <gtest/gtest.h>
+
+#include <cassert>
+
+#include "env.h"
+#include "monitor.h"
+
+int main(int argc, char **argv) {
+  Watchdog *watchdog = Watchdog::Create("/tmp/stacktrace.cvmfs_unittests");
+  assert(watchdog != NULL);
+  // watchdog->Spawn();
+  CvmfsEnvironment* env = new CvmfsEnvironment(argc, argv);
+  ::testing::InitGoogleTest(&argc, argv);
+  ::testing::FLAGS_gtest_death_test_style = "threadsafe";
+  ::testing::AddGlobalTestEnvironment(env);
+  int result = RUN_ALL_TESTS();
```

Comment:
Shouldn't this call something like `rc::check()`?  Or is this invocation part of RapidCheck's gtest module?

Reply:
RapidCheck doesn't have any test runner or test harness. `rc::check` is used to run individual test cases.

If it's integrated with GoogleTest, then the test runner part is specific to GTest and we define
our QuickCheck cases using `RC_GTEST_FIXTURE_PROP`.

I don't know, maybe I misunderstood your question.

Reply:
Thanks, that answers my question.